.github/workflows/ci.yaml

```diff
@@ -19,3 +19,8 @@ jobs:
           version: latest
           skip-build-cache: true
           skip-pkg-cache: true
+
+      - name: coverage
+        uses: codecov/codecov-action@v2
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
how I can include this secret in the project?

Reply:
Please go to https://github.com/rodrigo-brito/ninjabot/settings first. The Secrets tab on the left, and add a new repository secret. Use `CODECOV_TOKEN` for the name and the value for the token can be found at https://app.codecov.io/gh/rodrigo-brito/ninjabot/settings/general once you enable it on the codecov frontend.

Reply:
Ok, done